tests/components/hassio/test_init.py

```diff
@@ -51,8 +55,8 @@ async def test_setup_api_ping(hass, aioclient_mock):
         result = await async_setup_component(hass, "hassio", {})
         assert result
 
-    assert aioclient_mock.call_count == 6
-    assert hass.components.hassio.get_homeassistant_version() == "0.110.0"
+    assert aioclient_mock.call_count == 7
+    assert hass.components.hassio.get_core_info()["version_latest"] == "1.0.0"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)